include/scrimmage/simcontrol/SimUtils.h

```diff
@@ -92,16 +92,9 @@ boost::optional<std::string> run_test(const std::string &mission,
                                       const bool &init_python = true,
                                       const bool &finalize_python = true);
 
-bool logging_logic(MissionParsePtr mp, std::string s);
-
-std::shared_ptr<Log> preprocess_scrimmage(MissionParsePtr mp, SimControl &simcontrol);
```

Comment:
The logic in preprocess_scrimmage and postprocess_scrimmage were integrated into SimControl to reduce complexity of usage of SimControl